src/arrays/multidimensional_arrays/hard/q2/SimpleDisplay.java

```diff
@@ -0,0 +1,50 @@
+package arrays.multidimensional_arrays.hard.q2;
+
+/*
+Write a Java program called SimpleDisplay. It should have a main method which asks the user for the number of rows,
```

Comment:
Change question text to:
`Write a Java program called SimpleDisplay. It should have a main method asking the user for the number of rows and columns in an array and initialize an array of that size. Then, it should prompt the user to enter the values for this array. Also, create a method called even that accepts a multidimensional 2D array (int [][] nums) and counts the number of elements in it that are even. Print out this count in the method. Assume the user will provide integers greater than or equal to zero for the array row and column lengths. `